app/src/main/java/org/piwigo/helper/URLHelper.java

```diff
@@ -12,7 +12,9 @@ public URLHelper()
 
     public String getUrlWithMethod(String url, String method)
     {
-        url = url.replaceAll("https://", "").replaceAll("http", "");
+        url = url.toLowerCase()
+                .replaceAll("https://", "")
+                .replaceAll("http://", "");
```

Comment:
sorry to bother you again, but wouldn't this make the complete URL lower case? Shouldn't we make only the part before ":" (if any) to lower case?

Reply:
See https://github.com/Piwigo/Piwigo-Android/pull/209/commits/8ce29eda9a12aecd9d9527531a2887cc47f8c34f